src/main/java/se/liu/ida/hefquin/engine/queryplan/physical/impl/PhysicalOpRequestWithTranslation.java

```diff
@@ -55,10 +56,9 @@ public NullaryExecutableOp createExecOp( final ExpectedVariables ... inputVars )
 		else if ( fm instanceof TPFServer && req instanceof TriplePatternRequest ) {
 			return new ExecOpRequestTPFatTPFServerWithTranslation( (TriplePatternRequest) req, (TPFServer) fm );
 		}
-		// TODO: we need executable operators for the following cases
-//		else if ( fm instanceof BRTPFServer && req instanceof TriplePatternRequest ) {
-//			return new ExecOpRequestTPFatBRTPFServer( (TriplePatternRequest) req, (BRTPFServer) fm );
-//		}
+		else if ( fm instanceof BRTPFServer && req instanceof TriplePatternRequest ) {
+			return new ExecOpRequestTPFatBRTPFServerWithTranslation( (TriplePatternRequest) req, (BRTPFServer) fm );
+		}
```

Comment:
I think that the line below is incorrect. It is the same condition as above but, instead, it should cover the case in which req is a BindingsRestrictedTriplePatternRequest.

Reply:
This should be fixed now!